OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -17,48 +17,96 @@
 
 namespace OpenRA.Mods.Common.Activities
 {
-	public class Repair : Activity
+	public class Resupply : Activity
 	{
 		readonly IHealth health;
 		readonly RepairsUnits[] allRepairsUnits;
 		readonly Target host;
 		readonly WDist closeEnough;
 		readonly Repairable repairable;
+		readonly RepairableNear repairableNear;
+		readonly Rearmable rearmable;
+		readonly INotifyResupply[] notifyResupplies;
 
 		int remainingTicks;
-		bool played = false;
+		bool played;
+		bool paused;
+		bool repairComplete;
+		bool rearmComplete;
```

Comment:
Aren't these two bools redundant vs `activeResupplyTypes`? Just check if the appropriate flags are set.

Reply:
Yeah, they are now. Done.